dashboard/src/components/ListingItem/ListingItem.tsx

```diff
@@ -14,6 +15,7 @@ export interface IListingItem {
   hasBottomBorder?: boolean;
   showNumber?: boolean;
   onClick?: (item: string) => void;
+  tooltip?: boolean;
```

Comment:
It will be more flexible if the tooltip is the text itself, that way we can pass whatever we want to show